apps/minaintyg/src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -9,11 +9,7 @@ export function ProtectedRoute({ children }: { children: ReactNode }): JSX.Eleme
   }
 
   if (isError) {
-    if (import.meta.env.MODE === 'development') {
-      window.open('/welcome', '_self')
-    } else {
-      window.open('/saml2/authenticate/eleg', '_self')
-    }
+    window.open(import.meta.env.VITE_LOGIN_URL, '_self')
```

Comment:
Yeah, only variables with the prefix `VITE_` are available to the build process. This is to avoid accidentally leak variables from the system. https://vitejs.dev/guide/env-and-mode.html